test/e2e-v2/cases/pulsar/docker-compose.yml

```diff
@@ -122,7 +130,7 @@ services:
     command: bash -c "bin/apply-config-from-env.py conf/broker.conf && exec bin/pulsar broker"
 
   otel-collector:
-    image: otel/opentelemetry-collector:0.72.0
+    image: otel/opentelemetry-collector:0.50.0
```

Comment:
Why downgrade the version? Is something wrong for 0.72?

Reply:
It does not match the config in `otel-collector-config.yaml`.
```
  logging:
    logLevel: debug
```